lib/pages/Profile.dart

```diff
@@ -38,159 +24,171 @@ class Profile extends StatelessWidget {
+                                child: Container(
+                                  height: _picBtnDiameter,
+                                  width: _picBtnDiameter,
+                                  child: FittedBox(
+                                      child: Semantics(
+                                    label: 'Add Profile Picture',
+                                    child: FloatingActionButton(
+                                      backgroundColor: Colors.black,
+                                      child: Icon(Icons.add,
+                                          size: _picBtnDiameter),
+                                      onPressed: () async {
+                                        ImagePicker picker = ImagePicker();
+                                        PickedFile pickedFile =
+                                            await picker.getImage(
+                                                source: ImageSource.gallery,
+                                                maxHeight: 200,
+                                                maxWidth: 200);
+                                        Uint8List bytes =
+                                            await File(pickedFile.path)
+                                                .readAsBytes();
+                                        String base64Image =
+                                            base64Encode(bytes);
+                                        riderProvider.updateRiderPhoto(
+                                            AppConfig.of(context),
+                                            Provider.of<AuthProvider>(context,
+                                                listen: false),
+                                            base64Image);
+                                      },
+                                    ),
+                                  )),
+                                ),
+                                left: _picDiameter * 0.61,
+                                top: _picDiameter * 0.66)
+                          ],
+                        )),
+                    Padding(
+                        padding: EdgeInsets.only(bottom: 30),
+                        child: Stack(
+                          clipBehavior: Clip.none,
+                          children: [
+                            Row(
+                                crossAxisAlignment: CrossAxisAlignment.center,
+                                children: [
+                                  Text(riderProvider.info.fullName(),
+                                      style: TextStyle(
+                                        fontSize: 22,
+                                        fontWeight: FontWeight.bold,
+                                      )),
+                                ]),
+                            Positioned(
+                              child:
+                                  Text('Joined ' + riderProvider.info.joinDate,
```

Comment:
May want to check how this is formatted in backend, because at least for driver it seems like it's yyyy-mm-dd so this might need to be updated if rider is also changed

Reply:
I think as of now, I will keep it as is.